azure/vm.sh

```diff
@@ -610,24 +615,52 @@ fn azure_vm_backup_list_all(prefix) {
 # azure_vm_backup_delete deletes a backup. This function
 # will also remove the locks that prevents backups deletion.
 fn azure_vm_backup_delete(backup_resgroup) {
-
-	dellock <= _azure_vm_backup_get_nodelete_lock($backup_resgroup)
+	dellock  <= _azure_vm_backup_get_nodelete_lock($backup_resgroup)
 	readlock <= _azure_vm_backup_get_readonly_lock($backup_resgroup)
 
-	echo "backup delete: resource group: " + $backup_resgroup
-	echo "backup delete: removing lock: " + $dellock
+	echo "backup delete: resource group: "+$backup_resgroup
+	echo "backup delete: removing lock: "+$dellock
+
 	azure_lock_delete($dellock, $backup_resgroup)
-	echo "backup delete: removing lock: " + $readlock
+
+	echo "backup delete: removing lock: "+$readlock
+
 	azure_lock_delete($readlock, $backup_resgroup)
 
 	# WHY: Deleting locks is not synchronous, this is a VERY
 	# lame workaround while we don't fix the azure_lock_delete function.
 	lame_workaround_sleep = "5"
+
 	sleep $lame_workaround_sleep
-	echo "backup delete: locks removed, deleting resource group: " + $backup_resgroup
+	echo "backup delete: locks removed, deleting resource group: "+$backup_resgroup
+
 	azure_group_delete($backup_resgroup)
 }
 
+# azure_vm_backup_exists returns an empty string on success
+# (the resource group exists) or a non empty string with details
+# on what is wrong with the backup resource group.
+fn azure_vm_backup_exists(backup_resgroup) {
```

Comment:
@katcipis what do you think about returning a status as log with the error or empty string? It might be a stupid idea, I just thought it would look more like a command.